app/invocation/invocation_action_card.tsx

```diff
@@ -165,6 +165,9 @@ export default class InvocationActionCardComponent extends React.Component<Props
         if (operation.response?.result) {
           this.setState({ actionResult: operation.response.result });
         }
+        if (operation.done) {
```

Comment:
```suggestion
        // Fetch the full response from cache, since it contains
        // some additional metadata not sent on the stream.
        if (operation.done) {
```